src/web/components/Core/Dialog.scss

```diff
@@ -14,7 +14,7 @@
     background-color: var(--theme-background-content);
     border-radius: 5px;
     box-shadow: 0 4px 14px var(--theme-box-shadow);
-    max-width: 450px;
+    max-width: 37.875rem;
```

Comment:
Are you sure you want to change the max width of all dialogs? This makes the Add Team Member dialog look a bit big. Also why use rems here?